modules/users/default.nix

```diff
@@ -146,8 +146,8 @@ in
         homeDirectory=$(dscl . -read /Users/nobody NFSHomeDirectory)
         homeDirectory=''${homeDirectory#NFSHomeDirectory: }
 
-        if ! sudo dscl . -change /Users/nobody NFSHomeDirectory "$homeDirectory" "$homeDirectory" &> /dev/null; then
-          if [[ -n "$SSH_CONNECTION" ]]; then
+        if ! dscl . -change /Users/nobody NFSHomeDirectory "$homeDirectory" "$homeDirectory" &> /dev/null; then
```

Comment:
https://github.com/LnL7/nix-darwin/blob/ae406c04577ff9a64087018c79b4fdc02468c87c/modules/system/activation-scripts.nix#L99-L104

Checks still currently runs as part of user activation

Reply:
Ah, of course. I missed that this is part of the checks phase. Moved that commit back to the Plan stack, thanks!